packages/ui/src/stem-begroot-and-resource-overview/filter/select-tag-filter.tsx

```diff
@@ -33,12 +33,21 @@ const SelectTagFilter = forwardRef<HTMLSelectElement, Props>(
     if (!dataStore || !dataStore.useTags) {
       return <p>Cannot render tagfilter, missing data source</p>
     }
+    const randomId = Math.random().toString(36).substring(7);
+
+    function getRandomId(placeholder: string | undefined) {
+      if(placeholder && placeholder.length >= 1) {
+      return placeholder;
+      } else {
+      return randomId;
+      }
+    }
```

Comment:
Zelfde als in `multiselect-tag-filter`, misschien deze functie lostrekken naar eigen bestand?